aikau/src/main/resources/alfresco/renderers/_ActionsMixin.js

```diff
@@ -192,19 +192,19 @@ define(["dojo/_base/declare",
          if (this.mergeActions === true)
          {
             // Add the actions on the currentItem and then add additional custom actions...
-            this.currentItem.actions && array.forEach(this.currentItem.actions, lang.hitch(this, this.addAction));
-            this.customActions && array.forEach(this.customActions, lang.hitch(this, this.addAction));
+            lang.isArray(this.currentItem.actions) && array.forEach(this.currentItem.actions, lang.hitch(this, this.addAction));
```

Comment:
As a general rule, we use the "isArray" function of `alfresco/core/ObjectTypeUtils` (which is an alias for lang.isArray). The reason being is that it gives us the option at a future point to swap out the implementation for a more performant solution or even to go "native" should such as feature ever be available across all supported browsers.

However, I'm not going to hold this PR for that change.